packages/client-config/API.md

```diff
@@ -627,7 +627,7 @@ export type CustomClientConfig = {
 export const DEFAULT_CLIENT_CONFIG_VERSION: ClientConfigVersion;
 
 // @public
-export const generateClientConfig: <T extends "1" | "1.1" | "1.2" | "1.3" | "0">(backendIdentifier: DeployedBackendIdentifier, version: T, awsClientProvider?: AWSClientProvider<{
+export const generateClientConfig: <T extends "0" | "1" | "1.1" | "1.2" | "1.3">(backendIdentifier: DeployedBackendIdentifier, version: T, awsClientProvider?: AWSClientProvider<{

```

Comment:
This drift should be reverted (we have an edge case with extractor in our repo).
Just keep it out of PR, or run clean build. `npm run clean && npm install && npm run build && npm run update:api`

Sorry for inconvenience here.